docs/source/notebooks/Bias_correcting_climate_data.ipynb

```diff
@@ -15,11 +15,14 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "<PreparedRequest [GET]>\n",
-      "<PreparedRequest [GET]>\n"
+     "ename": "ImportError",
```

Comment:
There are errors in that cell is that intended?

Reply:
Some notebooks are yet not fixed. Since they're autodeployed, I'm hoping we can fix them later.